ci-operator/config/openshift/jenkins/openshift-jenkins-master.yaml

```diff
@@ -57,33 +69,126 @@ tests:
   commands: hack/check-plugins-supply-chain-change.sh
   container:
     clone: true
-    from: ocp_builder_rhel-8-golang-1.19-openshift-4.13
+    from: ocp_builder_rhel-8-golang-1.19-openshift-4.14
   skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
 - as: e2e-jenkins-image-tests
   commands: IMAGE_NAME=stable:jenkins make smoke
   container:
     clone: true
-    from: ocp_builder_rhel-8-golang-1.19-openshift-4.13
+    from: ocp_builder_rhel-8-golang-1.19-openshift-4.14
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: e2e-aws-openshift-4-14
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:latest
+      RELEASE_IMAGE_LATEST: release:latest
+    workflow: openshift-e2e-aws
+- as: e2e-aws-jenkins-client-plugin-openshift-4-14
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:latest
+      RELEASE_IMAGE_LATEST: release:latest
+    test:
+    - ref: jenkins-client-plugin-tests
+    workflow: ipi-aws
+- as: e2e-aws-jenkins-sync-plugin-openshift-4-14
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:latest
+      RELEASE_IMAGE_LATEST: release:latest
+    test:
+    - ref: jenkins-sync-plugin-e2e
+    workflow: ipi-aws
```

Comment:
I wonder if we should remove the `-4-14` from these and then we know they are for master branch, and then we just name the others? We will need to update this every release, but I don't' think that is a deal breaker.

Reply:
@divyansh42 ok, i removed the `-4-14` from the master branch tests.